src/embedded.js

```diff
@@ -348,7 +348,7 @@
             }
             if (typeof params['whiteLabelingOptions'] === 'object') {
                 this.whiteLabelingOptions = JSON.stringify(params['whiteLabelingOptions']);
-                this.whiteLabelingOptions = this.whiteLabelingOptions.replace('#', '');
+                this.whiteLabelingOptions = this.whiteLabelingOptions.replace(/#/g, '');
```

Comment:
The original regex was only replacing one instance of the hash, which ended up breaking cases where we're submitting multiple options.